.github/workflows/main.yml

```diff
@@ -0,0 +1,89 @@
+name: CLEO 5 Release Build
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
+
+jobs:
+  build:
+    runs-on: windows-2022
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.1
+
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+  
+      - name: Read Version Tag
+        id: read_version
+        run: node.exe .github/workflows/version.js
+
+      - name: Build Projects
+        shell: cmd
+        run: |
+          set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
+          msbuild -m CLEO5.sln /property:Configuration=Release /property:Platform=GTASA
+          msbuild -m cleo_plugins/CLEO_Plugins.sln /property:Configuration=Release /property:Platform=x86
+
+      - name: Prepare Files
+        id: prepare_archive
+        shell: cmd
+        run: |
+          @REM create output directory
+          mkdir .output\Release\cleo
+          mkdir .output\Release\cleo\cleo_modules
+          mkdir .output\Release\cleo\cleo_plugins
+          mkdir .output\Release\cleo\cleo_saves
+          mkdir .output\Release\cleo\cleo_text
+          mkdir .output\Release\cleo_readme
+
+          @REM copy files
+          copy third-party\bass\bass.dll .output\Release\bass.dll
+          copy source\cleo_config.ini .output\Release\cleo\.cleo_config.ini
+          copy cleo_plugins\.output\*.cleo .output\Release\cleo\cleo_plugins
+          copy cleo_plugins\.output\*.ini .output\Release\cleo\cleo_plugins
+
+          @REM install Silent's ASI Loader
+          curl https://silent.rockstarvision.com/uploads/silents_asi_loader_13.zip -o silents_asi_loader_13.zip
+          powershell.exe -NoP -NonI -Command "Expand-Archive '.\silents_asi_loader_13.zip' '.\.output\Release'"
+          move .output\Release\ReadMe.txt ".output\Release\cleo_readme\ASI Loader Readme.txt"
+          rmdir /s /q .output\Release\advanced_plugin_management_example
+          rmdir /s /q .output\Release\scripts
+
+
+      - name: Convert Markdown to HTML
+        id: md_to_html
+        run: |
+          npm install showdown
+          node.exe .github/workflows/markdown.js
+          move README.html .output\Release\cleo_readme\README.html
+          move CHANGELOG.html .output\Release\cleo_readme\CHANGELOG.html
+
+      - name: Pack binaries (Main)
+        uses: ThirteenAG/zip-release@master
+        with:
+          path: ./.output/Release/*
+          type: "zip"
+          filename: ${{ steps.read_version.outputs.archive_name }}
```

Comment:
What with `CLEO.SA_v` filename prefix?

Reply:
https://github.com/cleolibrary/CLEO5/blob/8375cc7b7a51abd82e97a33e2d82b34986f3d238/.github/workflows/version.js#L8

Reply:
Ok, but what about 'SA_v' part? Other GTA 3 and VC Cleo use this convention. It would be nice to have no collisions or confusion when downloading multiple versions.

Reply:
CLEO was originally made for SA. We don't call CLEO4 CLEO.SA.

Reply:
But at this point it is not only one CLEO in family. I already added platform info to readme as it starts to get problematic.

Reply:
I'm not forcing it but take in consideration that perhaps half of the CLEO userbase is younger than the game or even CLEO itself. That's no common knowledge 4-5 is for SA and 2.x is for 3 and VC. 
Uneducated user can download few files and then pick first "CLEO" one and put it in VC.
Just simple prefix to keep files piling in downloads directory organised.
`CLEO_SA_v5.0.0.zip` would be good too.

Reply:
How about SA.CLEO_vx.x.x.zip? TO be consistent with III.CLEO and VC.CLEO archives

Reply:
> Uneducated user can download few files and then pick first "CLEO" one and put it in VC.

which is why one of the goals for future CLEO versions is to merge SA and III/VC codebase and smooth the difference between those three games (just like what CLEO Redux did)

Reply:
> How about SA.CLEO_vx.x.x.zip? TO be consistent with III.CLEO and VC.CLEO archives
Hm ok then. I'm all for consistency.

Reply:
updated archive name https://github.com/x87/CLEO5/releases/tag/v5.0.0-alpha.16